File: android/versioned-abis/expoview-abi32_0_0/src/main/java/abi32_0_0/host/exp/exponent/ExponentPackageDelegate.java
package abi32_0_0.host.exp.exponent;

import java.util.List;

import abi32_0_0.expo.core.interfaces.Package;
import org.unimodules.core.interfaces.SingletonModule;
import abi32_0_0.host.exp.exponent.modules.universal.ExpoModuleRegistryAdapter;

public interface ExponentPackageDelegate {
  ExpoModuleRegistryAdapter getScopedModuleRegistryAdapterForPackages(List<Package> packages, List<SingletonModule> singletonModules);
}

File: android/versioned-abis/expoview-abi48_0_0/src/main/java/abi48_0_0/host/exp/exponent/ExponentPackageDelegate.kt
package abi48_0_0.host.exp.exponent

import abi48_0_0.expo.modules.core.interfaces.Package
import expo.modules.core.interfaces.SingletonModule
import abi48_0_0.host.exp.exponent.modules.universal.ExpoModuleRegistryAdapter

interface ExponentPackageDelegate {
  fun getScopedModuleRegistryAdapterForPackages(
    packages: List<Package>,
    singletonModules: List<SingletonModule>
  ): ExpoModuleRegistryAdapter?
}
